relational_database/homework.py

```diff
@@ -44,19 +50,30 @@ def task_3_list_customers_in_germany(cur) -> list:
 
     Returns: 11 records
     """
-    pass
+    cur.execute("""
+        SELECT * 
+        FROM Customers 
+        WHERE Country='Germany'
+        """)
+    return cur.fetchall()
 
 
 def task_4_update_customer(con):
     """
     Update first customer's name (Set customername equal to  'Johnny Depp')
     Args:
-        cur: psycopg cursor
+        con: psycopg cursor
```

Comment:
```suggestion
        con: psycopg connection object
```

взагалі дуже круто, що ти помітив помилку в описі :wink: :+1:

Reply:
точняк) спс